src/models/athb.js

```diff
@@ -0,0 +1,163 @@
+import { pmv_calculation } from "./pmv_ppd.js";
+
+/**
+ * Return the PMV value calculated with the Adaptive Thermal Heat Balance
+ * Framework [27]_. The adaptive thermal heat balance (ATHB) framework
+ * introduced a method to account for the three adaptive principals, namely
+ * physiological, behavioral, and psychological adaptation, individually
+ * within existing heat balance models. The objective is a predictive model of
+ * thermal sensation applicable during the design stage or in international
+ * standards without knowing characteristics of future occupants.
+ *
+ * This is a version that supports scalar arguments.
+ * @see {@link athb_array} for a version that supports arrays.
+ *
+ * @public
+ * @memberof models
+ * @docname Adaptive Thermal Heat Balance (athb)
+ *
+ * @param { number } tdb - dry bulb air temperature, in [°C]
+ * @param { number } tr - mean radiant temperature, in [°C]
+ * @param { number } vr - relative air speed, in [m/s]
+ *
+ * Note: vr is the relative air speed caused by body movement and not the air
+ * speed measured by the air speed sensor. The relative air speed is the sum of the
+ * average air speed measured by the sensor plus the activity-generated air speed
+ * (Vag). Where Vag is the activity-generated air speed caused by motion of
+ * individual body parts. vr can be calculated using the function jsthermalcomfort.utilities.v_relative.
+ *
+ * @param { number } rh - relative humidity, [%]
+ * @param { number } met - metabolic rate, [met]
+ * @param { number } t_running_mean - running mean temperature, in [°C]
+ *
+ * The running mean temperature can be calculated using the function
+ * jsthermalcomfort.utilities.running_mean_outdoor_temperature.
+ *
+ * @returns { number } athb_pmv - Predicted Mean Vote calculated with the Adaptive Thermal Heat Balance framework
+ *
+ * @example
+ * // Import the athb function from the jsthermalcomfort module
+ * import { athb } from "./models/athb.js";
```

Comment:
does matter right now but the examples should import from the public library

Reply:
deleted